app/models/dialog_import_validator.rb

```diff
@@ -10,12 +11,22 @@ def determine_validity(import_file_upload)
     raise ImportNonYamlError
   end
 
+  def determine_json_validity(dialog)
```

Comment:
maybe just determine_dialog_validity  for the method, and raise ParsedNonDialogError message below to get rid of JSON. 

btw, why do we need this wrapper around validate_dialog ?